keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/CloudDriverCache.kt

```diff
@@ -33,7 +32,6 @@ interface CloudDriverCache {
   fun credentialBy(name: String): Credential
   fun defaultKeyPairForAccount(account: String) =
     credentialBy(account).attributes["defaultKeyPair"] as String
-  fun namedImages(imageName: String, account: String): List<NamedImage>
```

Comment:
Took this out of `CloudDriverCache` as we handle it directly in the constraint evaluator